tests/schema/tests.py

```diff
@@ -532,6 +532,23 @@ def test_alter_text_field(self):
         with connection.schema_editor() as editor:
             editor.alter_field(Note, old_field, new_field, strict=True)
 
+    @skipUnlessDBFeature('can_defer_constraint_checks', 'can_rollback_ddl')
+    def test_alter_fk_checks_deferred_constraints(self):
+        """
+        #25492 - Altering a foreign key's structure and data in the same
+        transaction should work on backends supporting it.
+        """
```

Comment:
I agree that there isn't much value in adding "should work" all over the place. I was trying to stick to the existing convention but I think avoiding these is in line with the removal of "Test that" and friends.